specification/billing/resource-manager/Microsoft.Billing/preview/2018-11-01-preview/billing.json

```diff
@@ -4060,6 +4121,43 @@
         }
       }
     },
+    "BillingProfileCreationParameters": {
+      "description": "The parameters for creating a new billing profile.",
+      "properties": {
+        "displayName": {
+          "description": "The billing profile name.",
+          "type": "string"
+        },
+        "poNumber": {
+          "description": "Purchase order number.",
+          "type": "string"
+        },
+        "address": {
+          "description": "Billing address.",
+          "$ref": "#/definitions/Address"
+        },
+        "invoiceEmailOptIn": {
+          "description": "If the billing profile is opted in to receive invoices via email.",
+          "type": "boolean"
+        },
+        "enableAzureSkuIds": {
```

Comment:
Where is someone going to get these ids from? Presumably by reading the EnabledAzureSkus property on an existing BillingProfile? If so, should this parameter use the same type, so a caller doesn't need to convert from EnabledAzureSku[] to string[]?

Reply:
updated to use same type of EnabledAzureSkus for request as well.